internal/cli/atlas_alertconfig_create.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2020 MongoDB Inc
```

Comment:
[nit] naming should be `atlas_alert_config_create.go`

Reply:
I am also going to rename atlas_alertconfig_list